rpm/stackdriver-agent

```diff
@@ -25,7 +25,7 @@ RETVAL=0
 ARGS=""
 prog="stackdriver-collectd"
 pidfile=/var/run/stackdriver-agent.pid
-CONFIG=/opt/stackdriver/collectd/etc/collectd.conf
+CONFIG=/etc/stackdriver/collectd.conf
```

Comment:
Similar to deb/debian/stackdriver-agent.init, is it worth considering making this conditional - i.e., that we look in both the new and old locations and use the config file we find in one of those places (with preference for using the new location)?

Reply:
See comment in deb/debian/stackdriver-agent.init

Reply:
SGTM.